components/org.wso2.identity.outbound.adapter.websubhub/src/test/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtilTest.java

```diff
@@ -70,7 +70,8 @@
 public class WebSubHubAdapterUtilTest {
 
     private static final String WEBSUB_HUB_BASE_URL = "https://test.com/websub/hub";
-    private static final String TEST_OPERATION = "subscribe";
+    private static final String TEST_OPERATION_SUBSCRIBE = "register";
```

Comment:
Shall we use the `REGISTER` and `DEREGISTER` constants from the `WebSubHubAdapterConstants` class here?

Reply:
Fixed in df4ef35dea3ba71df0606cc97f36d6bddedce92b